h5bp/ssl/policy_future.conf

```diff
@@ -0,0 +1,43 @@
+# ----------------------------------------------------------------------
+# | SSL policy - Future                                                |
+# ----------------------------------------------------------------------
+
+# For services that want to be on the bleeding edge, the parameters
+# below sacrifice compatibility for the highest level of security & performance
+#
+# (!) This policy enfore a strong SSL configuration, which may raise
+#     errors with old clients.
+#     If a more compatible profile is required, use intermediate policy.
```

Comment:
This warning may be rewritten a bit to warn more about nginx version requirements instead.